Gemfile.lock

```diff
@@ -499,38 +503,38 @@ GEM
     omniauth-rails_csrf_protection (1.0.1)
       actionpack (>= 4.2)
       omniauth (~> 2.0)
-    pagy (3.6.0)
-    parallel (1.19.2)
-    parallel_tests (2.21.1)
+    pagy (3.14.0)
+    parallel (1.24.0)
```

Comment:
### [parallel](https://github.com/grosser/parallel)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.19.2 → 1.24.0

[_~~change-log~~, [source-code](https://github.com/grosser/parallel/tree/v1.24.0), [gem-diff](https://my.diffend.io/gems/parallel/1.19.2/1.24.0)_]





<details>
<summary>Commits</summary>

A change of **67** commits. See the full changes on [the compare page](https://github.com/grosser/parallel/compare/v1.19.2...v1.24.0).

These are the first 10 commits:
- (da5583d) [fix ci](https://github.com/grosser/parallel/commit/da5583db71feef04c012c683161421bc08b7e728)
- (ba67d62) [bump AR](https://github.com/grosser/parallel/commit/ba67d627c8e66568aa7e7b78f97f976410e364f2)
- (d191db7) [bump ruby requirements](https://github.com/grosser/parallel/commit/d191db712917a13c0beee3bc6b6c0a45b8181521)
- (ceaf691) [bump rails](https://github.com/grosser/parallel/commit/ceaf69192188f84db72a84e6de06a616ef9cf3cf)
- (e92cfcb) [bump rake to fix warnings](https://github.com/grosser/parallel/commit/e92cfcb817e22c4308825819fba4d146f9c5b889)
- (43c1949) [fix errors](https://github.com/grosser/parallel/commit/43c19496be4e5866529162827ed2738514486738)
- (7dd99a5) [remove cert](https://github.com/grosser/parallel/commit/7dd99a54c1924eff62eeb6c388a788d16c6b23f1)
- (6d2a85f) [Merge pull request #278 from grosser/grosser/ci](https://github.com/grosser/parallel/commit/6d2a85f1f04321a77da82dd702acf7786db0c7c9)
- (4c16b35) [allow breaking with value](https://github.com/grosser/parallel/commit/4c16b355eca801ff00598e509306c24eeb6c2338)
- (ea04bff) [Merge pull request #285 from grosser/grosser/break](https://github.com/grosser/parallel/commit/ea04bff5239c6aa63ec5976001bdb23ae2031873)


</details>